apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/ApolloAnnotationProcessor.java

```diff
@@ -22,11 +32,19 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class ApolloAnnotationProcessor implements BeanPostProcessor, PriorityOrdered {
+  private Pattern pattern = Pattern.compile("\\$\\{(.*)\\}:?(.*)");
+  private static Multimap<String, SpringValue> monitor = LinkedListMultimap.create();
+  private Logger logger = LoggerFactory.getLogger(ApolloAnnotationProcessor.class);
+
+  public static Multimap<String, SpringValue> monitor() {
```

Comment:
I think it may be better to add a new class called `SpringValueRegistry` and get it via singleton?